src/Components/Comments.jsx

```diff
@@ -3,13 +3,18 @@ import { fetchComments } from "../../utils";
 
 export const Comments = ({ review_id }) => {
   const [comments, setComments] = useState([]);
+  const [isLoadingComments, setIsLoadingComments] = useState(true);
 
   useEffect(() => {
     fetchComments(review_id).then(({ comments }) => {
       setComments(comments);
+      setIsLoadingComments(false);
     });
   }, []);
 
+  if (isLoadingComments) {
+    return <p>Comments are Loading...</p>;
+  }
```

Comment:
Your comments are stuck in a loading state with a 404 for review 14. Probably because it doesn't have comments. This shouldn't be an error.